src/MizarWidgets/SamplingWithFrameTrackInputWidgetTest.cpp

```diff
@@ -51,6 +59,44 @@ const std::vector<std::string> kThreadNamesSorted = {
     MakeThreadListItemString(kOtherThreadName, kOtherTid)};
 const QString kInputName = QStringLiteral("InputName");
 
+template <typename Container, typename AnotherContainer>
+auto MakeMap(const Container& keys, const AnotherContainer& values) {
```

Comment:
This name is rather generic. What about marking it static to avoid potential ODR problems.